addons/account/models/chart_template.py

```diff
@@ -95,10 +95,10 @@ def _get_chart_template_mapping(self, get_all=False):
         """
         # This function is called many times. Avoid doing a search every time by using the ORM's cache.
         # We assume that the field is always computed for all the modules at once (by this function)
-        field = self.env['ir.module.module']._fields['account_templates']
+        field = self.env['ir.module.module'].sudo()._fields['account_templates']
```

Comment:
You can easily check the use case in runbot to reproduce the error and check that the error happens for both changes.

Reply:
Can you put which is the error that happens on that case? I can't check in runbot, because the runbot is built for this code, not for the case where the sudo is not there.

Reply:
I'm afraid runbot has import options disabled:

![imagen](https://github.com/user-attachments/assets/5a78233e-5258-4080-b3f9-60bb8c338d3c)

and anyway I was talking about having the second sudo, but not the first one. Which error arises?

Reply:
Ok, you are right, trying to reproduce now the error again, the second sudo is not necessary.

Reply:
Sorry, really if it is necessary, an example of the error shown

```
File \"/opt/odoo/custom/src/odoo/odoo/http.py\", line 1782, in _serve_db\n    return service_model.retrying(self._serve_ir_http, self.env)\n  
File \"/opt/odoo/custom/src/odoo/odoo/service/model.py\", line 133, in retrying\n    result = func()\n  
File \"/opt/odoo/custom/src/odoo/odoo/http.py\", line 1809, in _serve_ir_http\n    response = self.dispatcher.dispatch(rule.endpoint, args)\n  
File \"/opt/odoo/custom/src/odoo/odoo/http.py\", line 2013, in dispatch\n    result = self.request.registry['ir.http']._dispatch(endpoint)\n  
File \"/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py\", line 221, in _dispatch\n    result = endpoint(**request.params)\n  
File \"/opt/odoo/custom/src/odoo/odoo/http.py\", line 757, in route_wrapper\n    result = endpoint(self, *args, **params_ok)\n  
File \"/opt/odoo/auto/addons/web/controllers/dataset.py\", line 24, in call_kw\n    return self._call_kw(model, method, args, kwargs)\n  
File \"/opt/odoo/auto/addons/web/controllers/dataset.py\", line 20, in _call_kw\n    return call_kw(request.env[model], method, args, kwargs)\n  
File \"/opt/odoo/custom/src/odoo/odoo/api.py\", line 468, in call_kw\n    result = _call_kw_multi(method, model, args, kwargs)\n  
File \"/opt/odoo/custom/src/odoo/odoo/api.py\", line 453, in _call_kw_multi\n    result = method(recs, *args, **kwargs)\n  
File \"/opt/odoo/auto/addons/base_import/models/base_import.py\", line 977, in parse_preview\n    fields_tree = self.get_fields_tree(self.res_model)\n  
File \"/opt/odoo/auto/addons/base_import/models/base_import.py\", line 318, in get_fields_tree\n    field_value['fields'] = self.get_fields_tree(field['relation'], depth=depth-1)\n  
File \"/opt/odoo/auto/addons/base_import/models/base_import.py\", line 318, in get_fields_tree\n    field_value['fields'] = self.get_fields_tree(field['relation'], depth=depth-1)\n  
File \"/opt/odoo/auto/addons/base_import/models/base_import.py\", line 289, in get_fields_tree\n    model_fields = Model.fields_get()\n  
File \"/opt/odoo/custom/src/odoo/odoo/models.py\", line 3437, in fields_get\n    description = field.get_description(self.env, attributes=attributes)\n  
File \"/opt/odoo/custom/src/odoo/odoo/fields.py\", line 852, in get_description\n    value = value(env)\n  
File \"/opt/odoo/custom/src/odoo/odoo/fields.py\", line 2807, in _description_selection\n    return determine(selection, env[self.model_name])\n  
File \"/opt/odoo/custom/src/odoo/odoo/fields.py\", line 102, in determine\n    return needle(*args)\n  
File \"/opt/odoo/auto/addons/account/models/company.py\", line 647, in _chart_template_selection\n    return self.env['account.chart.template']._select_chart_template(self.country_id)\n  
File \"/opt/odoo/auto/addons/account/models/chart_template.py\", line 114, in _select_chart_template\n    chart_template_mapping = self._get_chart_template_mapping()\n  
File \"/opt/odoo/auto/addons/account/models/chart_template.py\", line 101, in _get_chart_template_mapping\n    or self.env['ir.module.module'].search([])\n  
File \"/opt/odoo/custom/src/odoo/odoo/models.py\", line 1618, in search\n    return self.search_fetch(domain, [], offset=offset, limit=limit, order=order)\n  
File \"/opt/odoo/custom/src/odoo/odoo/models.py\", line 1641, in search_fetch\n    query = self._search(domain, offset=offset, limit=limit, order=order or self._order)\n  
File \"/opt/odoo/custom/src/odoo/odoo/models.py\", line 5407, in _search\n    model.check_access_rights('read')\n  
File \"/opt/odoo/custom/src/odoo/odoo/models.py\", line 4119, in check_access_rights\n    return self.env['ir.model.access'].check(self._name, operation, raise_exception)\n  
File \"/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_model.py\", line 2090, in check\n    raise AccessError(msg) from None\nodoo.exceptions.AccessError: 
```